jetty-server/src/main/java/org/eclipse/jetty/server/HttpInput.java

```diff
@@ -139,9 +143,23 @@ public int read(byte[] b, int off, int len) throws IOException
     {
         synchronized (_inputQ)
         {
+            long now=System.currentTimeMillis();
```

Comment:
I think it is too late for 9.x.y with regards to `System.currentTimeMillis()`, but 10.0.x is a perfect time to revisit this.  I have flagged #121 with the 10.0.x milestone